libraries/recipe_helpers.rb

```diff
@@ -18,10 +18,8 @@ def agent_version(node)
           dd_agent_version = dd_agent_version[platform_family]
         end
         if !dd_agent_version.nil? && dd_agent_version.match(/^[0-9]+\.[0-9]+\.[0-9]+((?:~|-)[^0-9\s-]+[^-\s]*)?$/)
-          if node['platform_family'] == 'suse' || node['platform_family'] == 'debian'
+          if %w[debian amazon fedora rhel suse].include?(node['platform_family'])
```

Comment:
Would it be more future-proof to just check if this is non-Windows here?

Reply:
It would, if we were adding another platform family derived from debian/rh/suse. On the other hand if we were adding a completely different platform family, it might cause hard to find bugs, so this goes either way. I'd probably prefer leaving it this way.